src/components/AnnotationViewer.vue

```diff
@@ -292,6 +311,82 @@ export default class AnnotationViewer extends Vue {
       this.drawNewConnections(displayedIds);
     }
     this.annotationLayer.draw();
+    this.drawTooltips();
+  }
+
+  drawTooltips() {
+    const oldFeatures = this.textLayer.features();
+
+    if (this.shouldDrawTooltips) {
+      // Avoid checking the store for each annotation
+      const all = this.tooltipOnAll;
+      const selected = this.tooltipOnSelected;
+      const hovered = this.tooltipOnHovered;
+
+      const displayedAnnotations = this.annotationLayer
+        .annotations()
+        .map((a: any) => a.options("storedAnnotation"))
+        .filter((a: any) => {
```

Comment:
Do we agree that in this case, "a" is what you called a "stored annotation" i.e. an annotation from the store?
If so, I guess instead of any, "a" type should be IAnnotation?

Reply:
It returns an `IAnnotation` or `undefined` if there is no "storedAnnotation" option, I'll change it for `IAnnotation | undefined`.